pygluu/kubernetes/gui/forms.py

```diff
@@ -0,0 +1,783 @@
+    Optional
+from wtforms.widgets import PasswordInput
+from pygluu.kubernetes.common import get_supported_versions
+from pygluu.kubernetes.settingdb import SettingDB
+settings = SettingDB()
+
+app_volume_types = {
+    "eks": {
+
+        "label": "Amazon Web Services - Elastic Kubernetes Service (Amazon EKS) MultiAZ - Supported",
+        "choices": [(6, "volumes on host"),
+                    (7, "EBS volumes dynamically provisioned"),
+                    (8, "EBS volumes statically provisioned")],
+        "default": 6
+    },
+    "gke": {
+        "label": "Google Cloud Engine - Google Kubernetes Engine",
+        "choices": [(11, "volumes on host"),
+                    (12, "Persistent Disk  dynamically provisioned"),
+                    (13, "Persistent Disk  statically provisioned")],
+        "default": 12
+    },
+    "aks": {
+        "label": "Microsoft Azure",
+        "choices": [(16, "volumes on host"),
+                    (17, "Persistent Disk  dynamically provisioned"),
+                    (18, "Persistent Disk  statically provisioned")],
+        "default": 17
+    },
+    "do": {
+        "label": "Digital Ocean",
+        "choices": [(21, "volumes on host"),
+                    (22, "Persistent Disk  dynamically provisioned"),
+                    (23, "Persistent Disk  statically provisioned")],
+        "default": 22
+    },
+    "local": {
+        "label": "Local Development",
+        "choices": [(26, "OpenEBS Local PV Hostpath")],
+        "default": 26
+    }
+}
+
+volume_types = {
+    "aks": {
+        "label": "Azure Options",
+        "choices": [("Standard_LRS", "Standard_LRS"),
+                    ("Premium_LRS", "Premium_LRS"),
+                    ("StandardSSD_LRS", "StandardSSD_LRS"),
+                    ("UltraSSD_LRS", "UltraSSD_LRS")],
+    },
+    "eks": {
+        "label": "AWS EKS Options",
+        "choices": [("gp2", "gp2"),
+                    ("io1", "io1"),
+                    ("st1", "st1"),
+                    ("sc1", "sc1")],
+    },
+    "gke": {
+        "label": "GCE GKE Options",
+        "choices": [("pd-standard", "pd-standard"),
+                    ("pd-ssd", "pd-ssd")],
+    }
+}
+
+
+def password_requirement_check(form, field):
+    regex_bool = re.match(
+        '^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*\W)[a-zA-Z0-9\S]{6,}$',
+        field.data)
+    if not regex_bool:
+        raise ValidationError(
+            "Password does not meet requirements. "
+            "The password must contain one digit, one uppercase "
+            "letter, one lower case letter and one symbol")
+
+
+class RequiredIfFieldEqualTo(Required):
+    """
+    a validator which makes a field optional if
+    another field has a desired value
+    """
+
+    def __init__(self, other_field_name, value, *args, **kwargs):
+        self.other_field_name = other_field_name
+        self.value = value
+        super(RequiredIfFieldEqualTo, self).__init__(*args, **kwargs)
```

Comment:
Use DataRequired here. I think that will work as Required is depreciated in WTforms 3.0